index.html

```diff
@@ -159,119 +172,126 @@ <h3>Multi-Post Stories <br> Gain+Glory</h3>
-    <section class="contact">
+
+    <section id="contact" class="contact">
         <div class="contactHeading">
             <h3>
                 I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-
             </h3>
         </div>
         <div class="contactForm">
-            <form action="https://formspree.io/f/xayzredb" method="POST">
+            <form action="https://formspree.io/f/moqzgvge" method="POST">
                 <ul>
                     <li>
                         <input name="fullName" type="text" placeholder="Full Name" maxlength="30" required>
                     </li>
                     <li>
-                        <input name="user_email" type="email" placeholder="Email address" required>
+                        <input name="email" type="email" placeholder="Email address" required>
                     </li>
                     <li>
-                        <textarea name="user_message" maxlength="500" placeholder="Hey I’d like to connect to..." required></textarea>
+                        <textarea name="messageBox" maxlength="500" cols="40" rows="5"
+                            placeholder="Hey I’d like to connect to..." required></textarea>
                     </li>
                     <li>
-                        <button type="submit" >Get in Touch</button>
+                        <button type="submit" value="">Get in Touch</button>
                     </li>
                 </ul>
             </form>
-            <p>cheptoopatience81@gmail.com</p>
-        </div> 
-    <div class="socialIcons">
-        <ul>
-            <li>
-                <a href="https://twitter.com/CheptooGyts">
-                    <img src="logo/twitter.png" alt="twitter">
-                </a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/in/cheptoo-patience-73499015a">
-                    <img src="logo/Linkedin icon.png" alt="Linkedin">
-                </a>
-            </li>
-            <li>
-                <a href="#"><img src="logo/medium.png" alt=""></a>
-            </li>
-            <li>
-                <a href="https://github.com/Cheptoo21">
-                    <img src="logo/github.png" alt="github">
-                </a>
-            </li>
-            <li>
-                <a href="#"><img src="logo/angellist.png" alt=""></a>
-            </li>
 
-        </ul>
-    </div>
+        </div>
+    </section>
+    <p class="contactEmail">cheptoopatience81@gmail.com</p>
     <div class="contactBackground">
-        <img src="logo/illustration-contact-form.png" alt=" ">
+        <img class="illustration1" src="logo/illustration-1-contact-form-desktop.svg" alt="Illustration1">
+        <img class="illustration2" src="logo/illustration-2-contact-form-desktop@2x.png" alt="Illustration2">
+        <img class="illustration3" src="logo/illustration-contact-form.png" alt="Illustration3">
+
     </div>
-</section>
+    <footer>
```

Comment:
Great job on your footer section! 👏👍 However, there are some updates needed for both the mobile and desktop versions.

- [ ] In the desktop version, there is a vertical line before the social icons in the design, but it's missing in your web page. Please update your webpage to include the vertical line as per the design. Additionally, the gap before and after the social icons is larger than in the design. Please reduce the gap to match the design specifications. Lastly, the background in your footer section does not match the design. Please update your code to match the background as closely as possible.
![Frame 3 (5)](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/55840999/56fa9ac1-2ac1-4bb7-b9f4-072235e0bdb7)
- [ ] In the mobile version, there is a significant gap between the email and social icons. Please update your code to reduce the gap and align the elements as per the design.
![Frame 5 (2)](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/55840999/d98ca07a-1046-4d9c-afbf-943fc2cd8229)
By addressing these issues, you'll create a footer section that aligns with the design and enhances the overall visual appeal of your web page. Keep up the good work! 💪✨